Design/js/mainscript.js

```diff
@@ -0,0 +1,78 @@
+const speakersArray = [
+  {
+    name: 'Paolo Razzini',
+    position: 'International Project Coordinator @ UNIDO',
+    description: 'Development Expert with a Extensive Experience in Programme Management.',
+    imageLink: 'Design/images/Paolo.jpg',
+    img1: 'Design/images/speaker-background.jpg',
+  },
+  {
+    name: 'Abenezer Tilahun',
+    position: 'Full-Stack Software Developer',
+    description: '3 years of Expericence of Full-Stack Software Developer and Remote pair-programming. ',
+    imageLink: 'Design/images/Abenezer.jpg',
+    img1: 'Design/images/speaker-background.jpg',
+  },
+  {
+    name: 'Temesgen Feseha',
+    position: 'Creative Hub Manager',
+    description: 'A Business and Tech Consultant with over 10 years Expericence  as a founder and a mentor for  start-up ecosystem .',
+    imageLink: 'Design/images/Temesgen.jpg',
+    img1: 'Design/images/speaker-background.jpg',
+  },
+  {
+    name: 'yohannes kanko',
+    position: 'Software Engineer @CBE',
+    description: 'Software Engineer at CBE, chief architect engineer responsible for DLSS algorithms in Nvidia graphics cards',
+    imageLink: 'Design/images/Jhon.jpg',
+    img1: 'Design/images/speaker-background.jpg',
+  },
+  {
+    name: 'Eden Eshetu',
+    position: 'Creative Hub Host',
+    description: 'Goal-oriented Events Assistant and part of the admin team with a clairvoyant eye for detail and proper team management with ample experience in working on projects, events, trainings .',
+    imageLink: 'Design/images/Eden.jpg',
+    img1: 'Design/images/speaker-background.jpg',
+  },
+  {
+    name: 'Abrham Yifter',
+    position: 'Senior System Developer @ Excellarate',
+    description: 'Over 4 years of well experienced  as System Developer at Spreedly building a PCI compliant API.',
+    imageLink: 'Design/images/Abrham.jpg',
+    img1: 'Design/images/speaker-background.jpg',
+  },
+];
+
+const speakersDOMElement = document.querySelector('.speakers');
+(() => {
+  speakersDOMElement.innerHTML = '';
+  let displayProps = '';
+  for (let i = 0; i < speakersArray.length; i += 1) {
+    speakersDOMElement.innerHTML = `${speakersDOMElement.innerHTML}
+    <li class="col-12 col-md-6 ${displayProps}">
```

Comment:
- [ ] Kindly take a look at the speaker's section, which is not being properly shown in smaller than 400px screens. It is probably because of the sizes that you used for the container of the section, which makes the image of the speaker to be overlapped with the content itself.
Here's an image for reference:
- 400px screen (or bigger) everything shows good.
![Screenshot_2022-02-09_19-13-14](https://user-images.githubusercontent.com/59100477/153264819-b0277b0c-0b69-41c5-ab87-c7f10a2443da.png)
- 370px screens (or smaller) images are overlaped.
![Screenshot_2022-02-09_19-13-33](https://user-images.githubusercontent.com/59100477/153264910-6fcb4572-dcb2-404a-8d08-d372f40373b6.png)

Reply:
Ok I will improve it.